### PR TITLE
Add Instance Method slice_scatter to PyTorch Frontend 

### LINKED ISSUE #22584: slice_scatter

### PR DESCRIPTION
**Pull request by Salah-Sal**
PR Description

This PR introduces the slice_scatter method for scattering updates into a tensor. It also includes the necessary tests to ensure the method works as expected.

Related Issue

Close #22584

Checklist

 Did you add a function? (Yes, added slice_scatter method.)
 Did you add the tests? (Yes, tests for the slice_scatter method have been added.)
 Did you follow the steps we provided?
Socials:

[Optional: Provide your Twitter handle if you wish]

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -12,7 +12,8 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License..
-# ==============================================================================
+# this is a test comment
```
Comment (jieunboy0516) [inline setup.py:15]: could you please remove unnecessary changes from here 😃

File: ivy/functional/frontends/torch/tensor.py [hunk 2]
```diff
@@ -1492,6 +1495,12 @@ def tile(self, *reps):
             reps = reps[0]
         return torch_frontend.tile(self, reps)
 
+    def slice_scatter(self, updates, begin, size, axis):
+        slices = [slice(None)] * self.ndim
+        slices[axis] = slice(begin, begin + size)
+        self.ivy_array[tuple(slices)] = updates.ivy_array
+        return self
```
Comment (jieunboy0516) [inline ivy/functional/frontends/torch/tensor.py:1502]: Hi, the `slice_scatter` is not a In-place method so the `self.ivy_array` should not be changed.

**Comment by jieunboy0516** [comment]
Hi @Salah-Sal , sorry for the late reply as i was not feeling well last week 😥 I have reviewed your code and left 2 comments. Furthermore, I am not sure if we should wait for the method `slice_scatter` to get implemented in `creation_ops.py` first, as i see there are `torch.slice_scatter` and `tensor.slice_scatter` as well. I will come back to you in a while, Thanks for your contribution to ivy!

**Comment by Sarvesh-Kesharwani** [comment]
Closing the PR for the time being, since there has been no activity for more than 2 weeks.
